spec/features/user_auth_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe "the user authorization feature", type: :feature do
+  let(:valid_user) { User.create(email: "paul@mail.com", full_name: 'Paul', password: "paul") }
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.